src/BroadcastSchedule.php

```
@@ -169,6 +176,47 @@ public function getNextRadioBroadcast(?RadioBroadcast $after = null)
+     *
+     * Takes the broadcast the caller is rendering rather than looking it up again: a boundary
+     * crossed between the two calls would otherwise pair the old programme with the new one's
+     * full remaining runtime, leaving the client on a stale slot for its entire duration.
+     */
+    public function getRefreshAfter(?RadioBroadcast $current): int
+    {
+        if (!$current) {
+            return self::REFRESH_FALLBACK;
+        }
+
+        return max(1, $current->end->timestamp - Carbon::now(wp_timezone())->timestamp);
```

PHPStan (2.2.7)

[warning] Binary operation "-" between float|int|string and float|int|string results in an error.


(binaryOp.invalid)